front/app/components/EventCard/index.tsx

```diff
@@ -9,32 +8,56 @@ import EventInformation from './EventInformation';
 // services
 import { IEventData } from 'services/events';
 
-// utils
-import { getIsoDate } from 'utils/dateUtils';
+// hooks
+import useLocale from 'hooks/useLocale';
+import useProject from 'hooks/useProject';
 
 // style
 import styled from 'styled-components';
-import { defaultCardStyle } from 'utils/styleUtils';
+import { defaultCardStyle, defaultCardHoverStyle } from 'utils/styleUtils';
+
+// other
+import { getIsoDate } from 'utils/dateUtils';
+import { isNilOrError, isNil } from 'utils/helperUtils';
+import clHistory from 'utils/cl-router/history';
+import { setScrollToEventId } from 'containers/ProjectsShowPage/shared/events/scrollToEventState';
 
-const Container = styled.div`
+const Container = styled.div<{ clickable?: boolean }>`
   width: 100%;
   padding: 30px;
   display: flex;
   ${defaultCardStyle};
+  ${({ clickable }) => (clickable ? defaultCardHoverStyle : '')}
+  ${({ clickable }) => (clickable ? 'cursor: pointer;' : '')}
   box-shadow: none;
   border: solid 1px #ccc;
 `;
 
 interface InputProps {
   event: IEventData;
   className?: string;
+  id?: string;
   showProjectTitle?: boolean;
+  showLocation?: boolean;
+  showDescription?: boolean;
+  clickable?: boolean;
 }
 
 interface Props extends InputProps {}
 
 const EventCard = memo<Props>((props) => {
-  const { event, className, showProjectTitle } = props;
+  const { event, className, id, clickable, ...otherProps } = props;
+  const projectId = event.relationships.project.data.id;
+
+  const locale = useLocale();
+  const project = useProject({ projectId });
+
+  const onClick = () => {
+    if (isNilOrError(locale) || isNil(project)) return;
+
+    setScrollToEventId(event.id);
+    clHistory.push(`/${locale}/projects/${project.attributes.slug}`);
```

Comment:
I did consider this, but it would conflict with the recently introduced phase parameter for projects with timelines. Of course I could add logic to determine whether the parameter is a phase number (an integer) or a `event.id` containing dashes and stuff, and take different actions based on that (go to the correct phase for the former, and scroll down to the correct event for the latter). But that also seems a bit dodgy...

Reply:
I don't believe this applies here, my suggestion would be 
```
clHistory.push(
  `/${locale}/projects/${project.attributes.slug}?highlightEvent=${event.id}&pickPhase=${phaseId}`
);

// and then you retrieve the params in the projects page, with withRouter's query object

cont scrollToEventId = query?.highlightEvent;
```
Also neat that you can easily pick that up from the eventCard and apply some styling without adding a prop or anything